web/src/personal_menu_popover.js

```diff
@@ -40,6 +42,25 @@ export function initialize() {
                 appendTo: document.body,
             });
 
+            $popper.on("change", ".theme-option", (e) => {
+                const prev_theme_code = user_settings.color_scheme;
+                const $theme_input = $(e.currentTarget);
+                const new_theme_code = $theme_input.data("themeCode");
```

Comment:
`.attr("data-theme-code")` is a nicer way to write this without the conversion to camelCase.

Reply:
Used `.attr()` instead of `.data()`, thanks!